### DISCUSSION TITLE
Adding tick lines and labels to the x and y axis of a D3 chart in React Native

### DISCUSSION THREAD

**Discussion by wildman82** [d3/d3#3851] [Q&A] [answered]
Hello,

I've modified the chart from this tutorial:

https://betterprogramming.pub/d3-and-react-native-an-essential-guide-to-line-graphs-dc1ce392b440

I'm looking to add tick lines and labels to the x and y axis. Can anybody please provide some suggestions?

The code for the example is here:

https://github.com/friyiajr/D3LineChartSample/tree/main/src

Thanks!

**Comment by Fil** [comment, marked as answer]
To create axes you can use functions such as d3.axisBottom, d3.axisLeft etc. See https://d3js.org/d3-axis
